web/index.html

```diff
@@ -382,13 +383,13 @@
       </header>
       <div id="layer-modal-main">
         <div id="layer-categories">
-          
+
         </div>
         <div id="selected-category">
-          
+
```

Comment:
I agree. This is how it is currently on master, though I should have cleaned it up while I was working in this area. I'm not sure why git is picking this up as a change.